runelite-client/src/main/java/net/runelite/client/plugins/attackstyles/AttackStylesPlugin.java

```diff
@@ -164,6 +185,18 @@ private void processWidgets()
 				hideWidget(client.getWidget(widgetKey), widgetsToHide.get(equippedWeaponType, widgetKey));
 			}
 		}
+
+		equippedWeaponType = WeaponType.TYPE_NULL;
+
+		if (widgetsToHide.containsRow(equippedWeaponType))
```

Comment:
Can you document this entire new block? i have no idea why is it present and why  is it using TYPE_NULL and why is it duplicate of the previous block but also hiding autoretalliatetext.